packages/cubejs-jdbc-driver/src/JDBCDriver.ts

```diff
@@ -183,6 +183,14 @@ export class JDBCDriver extends BaseDriver {
       acquireTimeoutMillis: 120000,
       ...(poolOptions || {})
     }) as ExtendedPool;
+
```

Comment:
If you're already touching this driver. Can you fix lint warning above too? (can't pin my comment there as there are no changes at that lines)

Reply:
Skimmed through, fixed all the minor issues.
Well, one not-so-minor, I guess: `destroy` function in pool's factory didn't actually call `close` on connection, only promisified it.